### DISCUSSION TITLE
Phishing and Brand Impersonation Attack

### DISCUSSION THREAD

**Discussion by lidiia-p-pf** [vercel/community#6744] [Help] [answered]
### Summary

Hello,

The following malicious website, created on your platform, is clearly impersonating Pancakeswap and conducting a phishing attack: https://frontend2.vercel.app/ 

Considering its deliberately abusive nature, this web threat should be taken down immediately. 

refID: I7ZAeQYLOIn1eHwUxiMo

Please escalate this and speed up the resolution of the case.

Thank you.

### Steps to Reproduce

Please visit: https://frontend2.vercel.app/

<img width="939" alt="Evidence" src="https://github.com/vercel/community/assets/169062735/89671c65-4b6a-4c7b-b7ac-d35fa136a4bc">

**Comment by juste-n-pf** [comment]
Hey Vercel Team, @amyegan, please have a look at this, phishing attack is still available at https://frontend2.vercel.app/. It's been also reported by email on April 24, case number is 00223722 & 00223723. Please suspend it ASAP, thanks!

**Reply by milannshrestha** [reply]
cc @amyegan

**Reply by amyegan** [reply, marked as answer]
@lidiia-p-pf @juste-n-pf Thanks for reporting this. You've already done the right thing by sending a message to [abuse@vercel.com](mailto:abuse@vercel.com). Because both reports were about the same site, they have been joined into one ticket. You will receive an update on 00223722 when the team has finished their review and taken appropriate action.

https://vercel.com/guides/how-to-get-vercel-support

**Reply by milannshrestha** [reply]
Hi @amyegan, domain above and other cases from us has not be responded for suspension. Please expedite the takedown.
